File: dotnet-back-end/ktdi-app/Model/ReportDamage.cs
using Microsoft.AspNetCore.Authorization;
using Microsoft.AspNetCore.Mvc;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

[Authorize]
[ApiController]
[Route("api/[controller]")]
public class ReportDamage
{
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int ReportId { get; set; }

    public int UserId { get; set; }

    public string ImagePath { get; set; }

    public string TypeOfDamage { get; set; }

    public string Description { get; set; }

    public string Status { get; set; } = "Draft";

    [ForeignKey("UserId")]
    public Authentication User { get; set; }
}

File: springboot-back-end/src/main/java/com/model/ReportDamage.java
package com.model;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Table(name = "ReportDamage")
@Data
public class ReportDamage {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "report_id")
    private Long reportId;

    @ManyToOne
    @JoinColumn(name = "user_id", nullable = false)
    private Authentication user;

    @Column(name = "image_path")
    private String imagePath;

    @Column(name = "type_of_damage")
    private String typeOfDamage;

    private String description;

    private String status = "Draft";
}